README.md

```diff
@@ -1 +1,83 @@
-# git
\ No newline at end of file
+<!-- Headings -->
+# Sample .md file
+## This is a sub heading
+### Another sub heading 
+---
+
+<!-- italics-->
+_This text in italic_
+___
+
+<!--bold-->
+__This text in bold__
+___
+
+<!--strikethrough-->
+~~Strikethrough~~
+___
+
+<!--Horizontal Rule-->
+<!--- ___>
+
+<!--BlockQuote-->
+>This is a block quote
```

Comment:
Please experiment whether this can be extended over multiple lines.

Reply:
Done, extended over multiple lines